index.bs

```diff
@@ -11506,7 +11506,7 @@ sound.
 </figure>
 
 <h4 id="stereopanner-algorithm">
-StereoPannerNode Panning</h4>
+StereoPannerNode Panning (equal-power only)</h4>
```

Comment:
Don't think saying equal-power only adds anything.  And nowhere in the algorithm does it say equal-power.

Reply:
I mentioned it because it was mentioned in the link, but I'll remove both